commands/cli/parse.go

```diff
@@ -357,7 +358,7 @@ func appendStdinAsString(args []string, stdin *os.File) ([]string, *os.File, err
 }
 
 func appendFile(args []files.File, inputs []string, argDef *cmds.Argument, recursive bool) ([]files.File, []string, error) {
-	fpath := inputs[0]
+	fpath := filepath.ToSlash(filepath.Clean(inputs[0]))
```

Comment:
Are slashes invalid in windows filenames? I know the opposite isnt true, i can make a file in linux with `\` in the name. I'm not certain how things like that would play with `filepath.ToSlash` here.